src/ShareToken.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import firebase from './lib/firebase';
+import { db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { useHistory } from 'react-router-dom';
+import { withSnackbar } from 'notistack';
+
+const ShareToken = (props) => {
+  const [shareToken, setShareToken] = useState(null);
+
+  // 1. Make a on submit function where we pull list of collections
+  let history = useHistory();
+  const onSubmitHandler = (event) => {
+    event.preventDefault();
+    db.collection(shareToken)
+      .get()
+      .then((querySnapshot) => {
+        if (querySnapshot.empty) {
+          props.enqueueSnackbar('Incorrect token, please try again', {
+            variant: 'error',
+          });
+        } else {
+          props.enqueueSnackbar('Success', {
+            variant: 'success',
+          });
+          localStorage.setItem('userToken', shareToken);
+          history.push('/list');
+        }
+      });
+  };
+
+  // 2. Create change function that keeps track of user input
+  const onChangeHander = (event) => {
```

Comment:
```suggestion
  const onChangeHandler = (event) => {
```

Reply:
Wow, good eye @luisaugusto ! I had to stare at this for a full minute before I could see the typo 😅